event/happiness_egg.asm

```diff
@@ -181,9 +181,9 @@ DayCareStep:: ; 7282
 	dec hl
 	inc [hl]
 	ld a, [hl]
-	cp 5242880 / $10000
+	cp HIGH($500000 >> 8)
 	jr c, .check_egg
-	ld a, 5242880 / $10000
+	ld a, HIGH($500000 >> 8)
```

Comment:
`MAX_EXP_DAY_CARE`? I always assumed you could go all the way to 100 in the daycare.

Reply:
5,242,880 is more than a Pokémon can ever gain, but yeah, I'll add a constant.